examples/own_dataset/train.py

```diff
@@ -50,6 +54,26 @@ def __call__(self, atoms, adjs):
         return x
 
 
+class ScaledAbsError(object):
+
+    def __init__(self, scale='standardize', ss=None):
+        self.scale = scale
+        self.ss = ss
+
+    def __call__(self, x0, x1):
+        if isinstance(x0, Variable):
+            x0 = cuda.to_cpu(x0.data)
+        if isinstance(x1, Variable):
+            x1 = cuda.to_cpu(x1.data)
+        if self.scale == 'standardize':
+            scaled_x0 = self.ss.inverse_transform(cuda.to_cpu(x0))
```

Comment:
It raises a error when `self.ss` is `None`, which happens when the `ss` option is a default value.

Reply:
I removed scale option, only check scaler